### DISCUSSION TITLE
v3.2.1

### DISCUSSION THREAD

**Discussion by osy** [osy86/HaC-Mini#1025] [Releases]
## Changes (v3.2.0)
* Support macOS Ventura
* Add FeatureUnlock kext to support AirPlay mirroring on >= 12.3 (#991) (thanks @goodbest)
* Add AppleIGB kext for LAN2 on >= 12.0 (#947)
* Add timeout for Brcm wifi cards to fix boot issues on >= 12.1 (thanks @goodbest)
* Removed manual UI scale option (OpenCore now detects this automatically)
* Enabled OpenCore support for booting Linux directly

## Changes (v3.2.1)
* Moved AppleIGB to an optional install because it is causing kernel panics
* Secure Boot is now checked by default. This will require additional steps to use and *failing to do these steps will result in broken boot*. **Read the [previous release notes](https://github.com/osy/HaC-Mini/releases/tag/v3.1.5) for details.**

## Driver Updates (v3.2.0)
* AirportBrcmFixup 2.1.4 -> 2.1.6
* AppleALC 1.7.1 -> 1.7.3
* BrcmPatchRAM 2.6.1 -> 2.6.3
* FeatureUnlock 1.0.8 -> 1.0.9
* Lilu 1.6.0 -> 1.6.1
* NVMeFix 1.0.9 -> 1.1.0
* OpenCore 0.8.0 -> 0.8.2
* Polaris22Fixup v1.3.5 -> v1.3.6
* VirtualSMC 1.2.9 -> 1.3.0
* WhateverGreen 1.5.8 -> 1.6.0

## Driver Updates (v3.2.1)
* AppleALC 1.7.3 -> 1.7.5
* BrcmPatchRAM 2.6.3 -> 2.6.4
* IntelBluetoothFirmware 2.1.0 -> 2.2.0
* Lilu 1.6.1 (OOB) -> 1.6.2
* OpenCore 0.8.3 (OOB) -> 0.8.5
* WhateverGreen 1.6.0 -> 1.6.1

## Known Issues
* OpenIntelWireless does not have a stable build for Ventura and therefore is not included in the package

## Messages
* **Important**: Apple has started requiring Secure Boot for updating to macOS Monterey 12.2 and above (including to macOS Ventura). See the [previous release](https://github.com/osy/HaC-Mini/releases/tag/v3.1.5) for details.
* [As announced a few months ago](https://github.com/osy/HaC-Mini/discussions/578), this project will be transitioning to a long-term support model. macOS Ventura will likely be the last supported version.

<hr /><em>This discussion was created from the release <a href='https://github.com/osy/HaC-Mini/releases/tag/v3.2.1'>v3.2.1</a>.</em>

**Comment by SpookHCK** [comment]
I'm not sure this works on macOS Ventura; there is a new issue opened; the installer seems to fail on 13.0 both clean install and upgrade from 12.6.1

**Reply by osy** [reply]
I've seen it; you don't have to post it again. I don't have free time currently but will take a look as soon as I do.
